spec/latest/core/index.html

```diff
@@ -164,26 +167,7 @@
     granularity.</p>
 </section>
 
-<section>
-  <h2>Conformance</h2>
-
-  <p>This specification describes the conformance criteria for
-    <dfn data-lt="Hydra API documentation">Hydra API documentations</dfn>
-    and <dfn data-lt="Hydra client">Hydra clients</dfn>. These criteria are
-    relevant to authors, authoring tool implementers, and client
-    implementers. All authoring guidelines, diagrams, examples, and notes
-    in this specification are non-normative, as are all sections
-    explicitly marked as non-normative. Everything else in this
-    specification is normative.</p>
-
-  <p class="issue">Conformance for Hydra clients should probably not be
-    specified in this document.</p>
-
-  <p class="issue">Add normative statements</p>
-
-  <p>The key words MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD,
-    SHOULD NOT, RECOMMENDED, NOT RECOMMENDED, MAY, and OPTIONAL in this
-    specification have the meaning defined in [[!RFC2119]].</p>
+<section id="conformance" data-include="section/conformance.html">
```

Comment:
Forgive me saying, but this is just whining :). How problematic is it to use [local-web-server](https://www.npmjs.com/package/local-web-server) or [http-server](https://www.npmjs.com/package/http-server)?